docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -18,6 +18,7 @@ This aligns to the workflow shown in the [deployment architecture](../architectu
 It assumes the [Azure requirements](../requirements_data_azure.md#azure) are in place, including:
 
 * Azure subscription and service principal
+* If you want to provision infrastructure within a private network, then the Azure spoke virtual network and subnet for private endpoints must be provisioned for each environment. The hub network must contain a self-hosted agent. Please refer [Hub-Spoke network topology](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/hub-spoke?tabs=cli) for more details on Hub-spoke network topology in Azure.
```

Comment:
Does it have to use a hub-and-spoke topology, or is that just what we recommend?

Reply:
yes, we have to use hub-and-spoke topology only.